ConfusingNumberii.java

```diff
@@ -0,0 +1,54 @@
+// "static void main" must be defined in a public class.
+public class Main {
+
+    //Time Complexity: 0(5^l) where l is the no. of digits in n
```

Comment:
Redefine the TC here

Reply:
the time complexity is 5 to the power l, because I am checking for 5 confusing no's for n numbers and l is the no. of digits in the number

Reply:
Seems to be a correct logic to me .what say @jeelpatel18 ?

Reply:
you're welcome Jeel